src/ecrc-frontend/src/components/page/applicationForm/ApplicationForm.test.js

```diff
@@ -826,4 +825,360 @@ describe("ApplicationForm Component", () => {
       "/criminalrecordcheck/transition"
     );
   });
+
+  test("Screen is scrolled if birth location field is empty", async () => {
+    const incompleteApplicant = {
+      ...applicant,
+      birthPlace: "",
+      driversLicNo: "123456",
+      phoneNumber: "1234567890",
+      emailAddress: "bob@ross.com",
+      jobTitle: "Painter",
+      organizationFacility: "PBS"
```

Comment:
do all these need to be specified everytime?
or can you declare `incompleteApplicant` at the top and then just modify the one field you wish to every time?

Reply:
would allow us to reduce a lot of duplication redeclaring the similar fields in every test